core/src/main/resources/filodb-defaults.conf

```diff
@@ -243,6 +246,11 @@ akka {
       "filodb.query.LogicalPlan" = kryo
     }
 
+    default-dispatcher.fork-join-executor {
+      parallelism-factor = 2.0
+      parallelism-max = 32
```

Comment:
Why are we hard coding these values ? These should  be chosen based on the hardware the process is running on.

Reply:
We don't need as many threads for the default fork-join pool since it is now doing next to nothing.  Will document.